VisualStudio.gitignore

```diff
@@ -80,6 +80,8 @@ ipch/
 *.opensdf
 *.sdf
 *.cachefile
+*.VC.db
+*.VC.VC.opendb
```

Comment:
@shiftkey If we can be more specific, why not to? It is always `*.VC.VC.opendb` for temporary opendb created by Visual Studio. What if user will create its own `*.opendb` which must be in his repo?